src/startupscriptgenerator/src/common/buildManifestHelper.go

```diff
@@ -47,3 +47,8 @@ func GetBuildManifest(appPath string) BuildManifest {
 
 	return _buildManifest
 }
+
+func ManifestFileExists(appPath string) bool {
+	tomlFile := filepath.Join(appPath, consts.BuildManifestFileName)
+	return FileExists(tomlFile)
```

Comment:
in general, it's good to store the function return to a variable and then return the variable to facilitate debugging when needed